### ISSUE TITLE
Branch 2.0.0 doesn't compile

### ISSUE THREAD

**Issue by bendem** [jkcclemens/bins#36]
```
   Compiling bins v2.0.0 (file:///Users/benjamin/projects/github/bins)
error[E0277]: the trait bound `config::Config: serde::de::Deserialize<'_>` is not satisfied
   --> src/main.rs:610:3
    |
610 |   toml::from_str(&content).map_err(BinsError::Toml)
    |   ^^^^^^^^^^^^^^ the trait `serde::de::Deserialize<'_>` is not implemented for `config::Config`
    |
    = note: required by `toml::from_str`

error: aborting due to previous error

error: failed to compile `bins v2.0.0 (file:///Users/benjamin/projects/github/bins)`, intermediate artifacts can be found at `/Users/benjamin/projects/github/bins/target`

Caused by:
  Could not compile `bins`.

To learn more, run the command again with --verbose.
```

**Comment by jkcclemens** [comment]
TOML is probably using new serde. You can pin TOML's version to before they updated. It should compile that way. I'll update bins when I get home to fix this.
On Sunday, April 30, 2017 at 17:27 bendem <notifications@github.com> wrote:

Compiling bins v2.0.0 (file:///Users/benjamin/projects/github/bins) error[E0277]: the trait bound `config::Config: serde::de::Deserialize<'_>` is not satisfied --> src/main.rs:610:3 | 610 | toml::from_str(&content).map_err(BinsError::Toml) | ^^^^^^^^^^^^^^ the trait `serde::de::Deserialize<'_>` is not implemented for `config::Config` | = note: required by `toml::from_str` error: aborting due to previous error error: failed to compile `bins v2.0.0 (file:///Users/benjamin/projects/github/bins)`, intermediate artifacts can be found at `/Users/benjamin/projects/github/bins/target` Caused by: Could not compile `bins`. To learn more, run the command again with --verbose.

**Comment by jkcclemens** [comment]
This should be fixed by 4cf4be75023d1bf53640537d4be589210c9350a5. I'm unsure about other services besides gist at the moment.

**Comment by bendem** [comment]
Yep, working fine now. Thanks
